### PR TITLE
Update PAYLOAD.md Formatting

### PR DESCRIPTION
**Pull request by TillaTheHun0**
Just fixing some inconsistent indentation in some code blocks in `PAYLOAD.md` documentation.

### CODE REVIEW

File: docs/PAYLOAD.md [hunk 1]
```diff
@@ -1513,7 +1513,7 @@ However if you want your `on('notification')` event handler called but no notifi
 		"moredata": "Do more stuff",
 		"content-available": 1
 	},
-  "notId": 2 // unique ID you generate
+    "notId": 2 // unique ID you generate
```
Comment (fredgalvao) [inline docs/PAYLOAD.md]: This one doesn't seem to be 100% consistent with the surroundings. Am I seeing things?

Comment (TillaTheHun0) [reply docs/PAYLOAD.md]: hmm I see what you mean with the content above it. In my editor and in the markup, all of the indentation lines up. When I redid the indent, git indicated some additional changes on these lines, even though it 'looked' the same. I added it to the commit. Maybe some weird thing with spaces vs. indents?

Comment (fredgalvao) [reply docs/PAYLOAD.md]: image

**Comment by fredgalvao** [comment]
This seems fine/consistent now. Thanks for helping keep it up!

**Comment by TillaTheHun0** [comment]
Thank _you_ for maintaining! 👍

**Pull request merged**